src/js/keyboardInput.js

```diff
@@ -198,6 +207,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
     gpii.firstDiscovery.keyboardInput.registerKeypressListener = function (that, input, keymap) {
         input.keypress(function (e) {
             e.preventDefault();
+            e.stopPropagation();
```

Comment:
I believe the compound of these effects can be had via "return false"
http://stackoverflow.com/questions/1357118/event-preventdefault-vs-return-false